recipes/xcdf/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "XCDF" %}
+{% set version = "v3.00.03" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jimbraun/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 4e445a2fea947ba14505d08177c8d5b55856f8411f28de1fe4d4c00f6824b711
+
+build:
+  number: 0
+  skip: True  # [py3k or win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - zlib
+    - pyhton < 3
```

The version specification isn't needed as you have the `skip` above. You also probably need `python` as a runtime dependency:
```suggestion
    - python
  run:
    - python
```